build.gradle

```diff
@@ -49,13 +49,18 @@ ext {
 }
 
 dependencies {
+
+
+
   // Spring web
   implementation 'org.springframework.boot:spring-boot-starter-web'
 
   // Spring test
   testImplementation('org.springframework.boot:spring-boot-starter-test') {
     exclude group: 'org.junit.vintage', module: 'junit-vintage-engine'
   }
+  testImplementation 'junit:junit:4.12'
```

Comment:
The JUnit is still in pom. 
Spring version contains JUnit 5 in `org.springframework.boot:spring-boot-starter-test` artifact that is included in our pom